net/gce.sh

```diff
@@ -335,6 +343,9 @@ ec2|azure|colo)
   if [[ "$customMemoryGB" != "$defaultCustomMemoryGB" ]]; then
     usage "--custom-memory-gb only supported on cloud provider: gce"
   fi
+  if [[ "$profile" = "true" ]]; then
+    usage "--profile only supported on cloud provider: gce"
```

Comment:
what makes this gce-only?

note that we do the sneaky conditional on `$0` and symlink to this script with different names thing at line 8 above

Reply:
ok, i think if they're incompatible (and first, we actually start using them again!), then the right thing to do is fix perf there before we need it. i think it's safe to drop this gate

Reply:
> ok, i think if they're incompatible (and first, we actually start using them again!), then the right thing to do is fix perf there before we need it. i think it's safe to drop this gate

good, if we need other clouds, I will check on demand that perf works there as well